haiku-apps/wakeup/wakeup-1.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="An application that plays a sound after a given time"
+DESCRIPTION="WakeUp is a timer application that allows users \
+to set their own ringtone, giving them a more personalized \
+experience. This application could be used for many purposes, \
+like an alarm clock or a cooking timer, just to name a few."
+HOMEPAGE="https://github.com/HaikuArchives/WakeUp"
+COPYRIGHT="1999 Jonathan Villemure"
+LICENSE="MIT"
+REVISION="1"
+gitSrcRevision="116afedea73ba9779d3d641028b65fdb2dc19012"
+SOURCE_URI="https://github.com/HaikuArchives/WakeUp/archive/$gitSrcRevision.tar.gz"
+CHECKSUM_SHA256="28eb8504a2a436a117821aaa399b8c37f5363bd9fcb2ff814f6cb1b6264da311"
+SOURCE_DIR="WakeUp-$gitSrcRevision"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	wakeup = $portVersion
+	app:WakeUp = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+BUILD()
```

Comment:
Oops, you removed the blank line before BUILD... but we like to have one there.
(Sorry for being picky...)

Reply:
It's fine :) I still have a lot to learn